kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -109,6 +109,23 @@
       + "capture groups that are referenced in the format string using placeholders (i.e. $1) "
       + "(form of <topic regex>=<format string>)";
 
+  public static final String RECORD_ALIASES_CONFIG =                         "recordAliases";
```

Comment:
Can you please explain how we are going to use `recordAliases` to support topics with multiple schemas? 😄

Reply:
It's very probable that I just picked a silly name for this property.

Record names might be very long (think about fully qualified names for classes that are placed deep in the package structure) and we might want to make BigQuery table names short and clear for people who are not core dev team members (like data scientists).

Another purpose for this property was to filter the set of all schemas retrieved from schema registry to only those that are relevant (see `SchemaRetriever.getAllSchemas()`). Since we are going to get rid of `getAllSchemas()` it will be used only to provide a functionality of shortening table names.

All ideas for a better property name are welcome 👍

Reply:
Aha, now I see you why you named it `RECORD_ALIASES_CONFIG`. I think it is fair. 
If you ask for better name, maybe `RECORDS_TO_TABLE_POSTFIXES_CONFIG`?  (similar to `TOPICS_TO_TABLES_CONFIG`) What you think?

Reply:
It's longer but makes clear what's the purpose of the param 👍